stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/ChatDomainImpl.kt

```diff
@@ -754,7 +754,7 @@ internal class ChatDomainImpl internal constructor(
         }
 
         // store the channel configs
-        repos.configs.insert(configs)
+        repos.configChannel(configs)
```

Comment:
Why not `insertChannelConfig`?

Reply:
My though is that this method is used to configure the channel, that fact that the configuration is stored can be extracted away. But no problems with `insertChannelConfig` either, what do you think?

Reply:
I consider RepositoryHelper just as a database wrapper. ChatDomain is a key for logic. So I think straightforward insert is better than

Reply:
Changed